docs/README.md

```diff
@@ -201,6 +201,38 @@ versions:
   for Linux
 - [`nvm-windows`](https://github.com/coreybutler/nvm-windows) - Works with native Windows
 
+##### Install Git
+Booster will initialize a Git repository when you create a new project, so it is required that you have it already installed it in your system.
```

Comment:
What do you think about saying here that you can skip this with the corresponding flag? Suggestion:
```suggestion
Booster will initialize a Git repository when you create a new project (unless you use the `--skipGit` flag), so it is required that you have it already installed in your system.
```

(I also removed a duplicated "it" you had there 😉 )